backend/pyproject.toml

```diff
@@ -23,6 +23,7 @@ django-fsm = "^2.7.0"
 django-utils2 = "^2.8.0"
 pynacl = "^1.3.0"
 drf-writable-nested = "^0.6.0"
+django-reversion = "^3.0.7"
```

Comment:
This is _not_ actually necessary here, but I included it already because I think we'll use it soon.

I thought that I'll need to include versions in the elections model already, but it seems that it's not the case.
`django-reversion` allows us to version instances of our models easily. This means that if in v1.0.0 we have a `Vote` model that computes a signature using `field1`, `field2` and in v1.1.0 we add a `field3`, then by using `reversion` we can easily distinguish instances of elections that are still in v1.0.0 from these in v1.1.0.

So - in short - it's not yet necessary and I will remove it on your request, but on the other hand maybe it's good to choose an approach right now.

Reply:
This sounds like a cool tool and might be useful soon on prod. It's a good idea to become familiar with it